chapter02/worksheets/solution2.1.7.8.scala

```diff
@@ -0,0 +1,32 @@
+/** author: Andreas Röhler */
+
+/** 
+  Exercise 2.1.7.8
+
+  Write the solution of Exercise 2.1.7.7 as a function with type
```

Comment:
```
Exercise 2.1.7.8 Write the solution of Exercise 2.1.7.7 as a function with type parameters P and Q instead of the fixed types Int and String. The return type of the function should be Map[P, Q]. Run some tests using types P = Double and Q = Set[Boolean].
```